jaxrs/src/main/java/se/fortnox/reactivewizard/ExceptionHandler.java

```diff
@@ -72,7 +73,8 @@ public Publisher<Void> handleException(HttpServerRequest request, HttpServerResp
             webException = new WebException(BAD_REQUEST, "invalidjson", throwable.getMessage());
         } else if (throwable instanceof WebException we) {
             webException = we;
-        } else if (throwable instanceof ClosedChannelException || throwable instanceof AbortedException || throwable instanceof CancellationException) {
+        } else if (throwable instanceof ClosedChannelException || throwable instanceof AbortedException || throwable instanceof CancellationException
```

Comment:
Sorry for being vague, I meant extracting parts to variables or methods so that the conditions are in a concise and readable format.

Like for example
`if (isExpectedClientException) `

Reply:
Yes, that was a good suggestion. Fixed :)